content/post/Android_Root101.en.md

```diff
@@ -48,17 +49,20 @@ Place this file inside the folder created after extraction.
 
 > `cd..` is used to go down one directory
 
+Enable Developer Options on your phone and enable USB-Debugging. To know more [click-here](https://www.google.com/search?rlz=1C1CHBD_enIN850IN850&ei=5L_7XI0insTPuw-E0JiIAQ&q=how+to+enable+developer+option+in+android&oq=how+to+enable+developer+option+in+android&gs_l=psy-ab.3..35i39j0j0i20i263j0l4j0i30l3.2954.3254..4277...0.0..0.140.265.0j2......0....1..gws-wiz.......0i71j35i304i39j0i13j0i13i30.0E6WUqzOMPo)
```

Comment:
Can you zero in on a particular website for this information, having a Google link to point back for information is really unstable and could possibly break anytime?

Reply:
Sure, on it. 😄